benchmark/generateReport.js

```diff
@@ -0,0 +1,40 @@
+const { writeFileSync } = require('fs')
```

Comment:
Can you fix the name of this file to not be camel case?

Reply:
The problem is we cannot have `leran bootstrap` on CI.  If we can have `leran bootstrap`, then I can set up this on CI.